roles/cifmw_ceph_spec/templates/ceph_spec.yml.j2

```diff
@@ -28,7 +28,11 @@ service_id: mgr
 service_name: mgr
 service_type: mgr
 ---
-{{ cifmw_ceph_spec_data_devices }}
```

Comment:
@katarimanojk can you change this so that if the user provides `cifmw_ceph_spec_data_devices` that it uses it?

I think we should default the devices list the way you have it with a loop but that if a user provides the variable with specific block devices, then it should use those block devices.

Reply:
@fultonj agree with you, latest patch will address it.
if `cifmw_ceph_spec_data_devices` is passed by the user, spec will be generated using it.
otherwise it defaults to the logic that uses the devices list in a loop based on `cifmw_num_osds_perhost`